options/m_option.c

```diff
@@ -3580,7 +3521,7 @@ static int set_obj_settings_list(const m_option_t *opt, void *dst,
                     goto error;
                 entry->label = talloc_strdup(NULL, val->u.string);
             } else if (strcmp(key, "enabled") == 0) {
-                if (val->format != MPV_FORMAT_FLAG)
+                if (val->format != MPV_FORMAT_BOOL)
                     goto error;
                 entry->enabled = val->u.flag;
```

Comment:
Reading`u.flag` when it's a `MPV_FORMAT_BOOL`?

Reply:
I blame splitting the commits.